collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,26 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
```

Comment:
@cofyc Friendly ping about import `GetPersistentVolumeClaimClass` from client-go instead of copy&&paste. This needs to update the Godep. 

@brancz WDYT.